nais/topics.yml

```diff
@@ -0,0 +1,24 @@
+
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: privat-k9-produksjonsstyring-sak-v1
+  namespace: k9saksbehandling
+  labels:
+    team: k9saksbehandling
+spec:
+  pool: {{kafka-pool}}
+  config:
+    cleanupPolicy: delete  # delete, compact
+    minimumInSyncReplicas: 1
+    partitions: 3
```

Comment:
>1 partitsjoner vil funke med rekkefølge på eventer fordi nøkkelen til topic'en er behandlingid, sånn at eventer på samme behandlingid vil havne etter hverandre i samme partisjon?

Reply:
Det stemmer. Så lenge man oppgir en nøkkel skal kafka bruke hashfunksjon på nøkkelen for å avgjøre partisjonen. Tre partisjoner er litt unødvendig, så lenge vi bare har én consumer, men det er uansett ikke noe problem